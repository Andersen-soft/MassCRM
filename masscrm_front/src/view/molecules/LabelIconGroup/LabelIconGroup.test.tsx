import * as React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import { PermIdentity } from '@material-ui/icons';
import { LabelIconGroup } from './index';

describe('LabelIconGroup', () => {
  it('LabelIconGroup should render correctly', () => {
    expect(
      shallowToJson(
        shallow(<LabelIconGroup count='0' label='test' icon={PermIdentity} />)
      )
    ).toMatchSnapshot();
  });
});
