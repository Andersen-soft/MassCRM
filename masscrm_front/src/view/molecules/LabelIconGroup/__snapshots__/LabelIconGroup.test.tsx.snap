// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LabelIconGroup LabelIconGroup should render correctly 1`] = `
<div
  className="makeStyles-labelIcon-1 "
>
  <span
    className="makeStyles-labelIconText-2"
  >
    test
    :
  </span>
  <span
    className="makeStyles-labelIconCount-3"
  >
    0
  </span>
  <CommonIcon
    IconComponent={
      Object {
        "$$typeof": Symbol(react.memo),
        "compare": null,
        "type": Object {
          "$$typeof": Symbol(react.forward_ref),
          "render": [Function],
        },
      }
    }
  />
</div>
`;
